### ISSUE TITLE
can not import json

### ISSUE THREAD

**Issue by shellquery** [go-python/gpython#236]
File "/opt/homebrew/Cellar/python@3.9/3.9.20/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 339, offset 80
                raise TypeError(f'the JSON object must be str, bytes or bytearray, '

SyntaxError: 'invalid syntax'

**Comment by sbinet** [comment]
yeah, `gpython` is (quoting the `README`) _a part re-implementation, part port of the Python 3.4 interpreter in Go_.
IIRC, f-strings were not yet in (if memory serves, they were brought in CPython 3.6).

we'd be happy to review a PR adding this feature :}
